test/utils/EIP712Utils.ts

```diff
@@ -0,0 +1,131 @@
+import {
+    MessageTypeProperty,
+    MessageTypes,
+    TypedMessage,
+    signTypedData,
+    SignTypedDataVersion
+} from '@metamask/eth-sig-util';
+
+export interface ForwardRequest {
```

Comment:
We had a conversation with Juraj during a daily about where we can get or  define it. Its suppose that we need to get it from the typechain-types folder.

Reply:
Refactored!